bin/runex

```diff
@@ -41,5 +41,5 @@ fi
 "$ex_home"/bin/build
 
 export CLASSPATH="$EXAMPLES_JAR_PATH:$ex_home/conf:$ACCUMULO_HOME/lib/*:$CLASSPATH"
-java "org.apache.accumulo.examples.$1" ${*:2}
+java ${ACCUMULO_JAVA_OPTS} "org.apache.accumulo.examples.$1" ${*:2}
```

Comment:
I suggest removing it here, and changing the instructions to use `JAVA_TOOL_OPTIONS` to set the agent. That is a [standard environment option for Java executables](https://docs.oracle.com/javase/8/docs/platform/jvmti/jvmti.html#tooloptions). There's also a `_JAVA_OPTIONS` environment variable that works similarly, but I can't find documentation on it.

The problem with `ACCUMULO_JAVA_OPTS` is it's non-standard... just a convention that we have in Accumulo's own scripts. This isn't something we want to leak into the examples, though, because doing so might require an explanation about conventions we use internally, and may communicate behavioral expectations that we don't guarantee. Introducing it here would be its first appearance in the examples, and I think it can be avoided in favor of standard conventions.

```suggestion
java "org.apache.accumulo.examples.$1" ${*:2}
```

Reply:
I didn't know about `JAVA_TOOL_OPTIONS`, that's pretty neat. I have tested this change and it works. I'll push it up in another commit.

Reply:
I actually think this standard option is what we should use everywhere, but I think some people didn't like the fact that it printed the options to the console to inform the user. Personally, I think it's a small price to pay to get a standardized convention. Certainly, it's more than adequate for examples, regardless of what the rest of Accumulo does in its main environment / scripts.